app/src/main/java/org/oppia/app/topic/review/TopicReviewViewModel.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.app.topic.review
+
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModel
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.domain.topic.TopicController
+import javax.inject.Inject
+
+/**
+ * [ViewModel] for showing a list of topic summaries. Note that this can only be hosted in fragments that implement
```

Comment:
Please update the comment with proper information .
suggestion  : 
TopicReviewViewModel is a [ViewModel] for preparing and managing the data for showing a list of topic review

Reply:
Removed this file because actually I am not using this ViewModel.